service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/RepresentativeSubmittedEventControllerTest.java

```diff
@@ -46,7 +46,7 @@
 @OverrideAutoConfiguration(enabled = true)
 class RepresentativeSubmittedEventControllerTest extends AbstractControllerTest {
 
-    private static final String REPPRESENTATIVE_FULLNAME = "John Smith";
+    private static final String REPRESENTATIVE_FULLNAME = "John Smith";
```

Comment:
object mapper shouldn't be required in this test, is in the base class

Reply:
done